### DISCUSSION TITLE
Push/Pull Workflow

### DISCUSSION THREAD

**Discussion by brycegbrazen** [Autodesk/maya-usd#2118] [General] [answered]
Based on the mayaUSD v0.16.0 release notes, I see that there is work being done regarding a Push/Pull workflow. This seems to be some sort of on the fly converting between USD data and Maya data. Can anyone provide some information on what this workflow is currently and what the plans are for this?
![pushpull](https://user-images.githubusercontent.com/69012021/154571166-4057fac2-1beb-4634-84d7-0308e2ed49a4.PNG)

**Comment by womanyoyoyo-adsk** [comment, marked as answer]
Hi @brycegbrazen, 

We will be able to share more information about the workflow with you soon. Stay tuned!

**Reply by brycegbrazen** [reply]
Hi @womanyoyoyo-adsk,

Just checking in on this workflow, and if there has been new documentation / videos explaining more about it yet? Or is it still unreleased?

**Reply by womanyoyoyo-adsk** [reply]
Hi @brycegbrazen, I will get back to you shortly about this.

**Reply by womanyoyoyo-adsk** [reply]
Hi @brycegbrazen, please check out the workflow here: #2665 

Thanks for your patience! Any feedback would be appreciated.
